python/lsst/ts/externalscripts/maintel/parameter_march_comcam.py

```diff
@@ -118,6 +119,14 @@ async def take_images(
             reason="EXTRA" + ("" if self.reason is None else f"_{self.reason}"),
             program=self.config.program,
         )
+        self.log.info("Waiting for data to be ingested by OODS.")
+        for _ in range(9):
```

Comment:
I think I did this one but we should not have a hard-coded number here. Can you make this a class attribute?